Source/WebCore/platform/graphics/MediaPlayerPrivate.h

```diff
@@ -44,11 +44,16 @@ namespace WebCore {
 class VideoFrame;
 
 class MediaPlayerPrivateInterface {
-    WTF_MAKE_NONCOPYABLE(MediaPlayerPrivateInterface); WTF_MAKE_FAST_ALLOCATED;
```

Comment:
Why are we dropping these?

Reply:
Because adding `RefCounted` to the subclasses causes a compiler error, as RefCounted is also `WTF_MAKE_NONCOPYABLE`.